automated install/basic-install.sh

```diff
@@ -1001,6 +1001,38 @@ remove_old_dnsmasq_ftl_configs() {
     fi
 }
 
+remove_old_pihole_lighttpd_configs() {
+    local lighttpdConfig="/etc/lighttpd/lighttpd.conf"
+    local condfd="/etc/lighttpd/conf.d/pihole-admin.conf"
+    local confavailable="/etc/lighttpd/conf-available/15-pihole-admin.conf"
+    local confenabled="/etc/lighttpd/conf-enabled/15-pihole-admin.conf"
+
+
+    if [[ -d "/etc/lighttpd/conf.d" ]]; then
+        if grep -q -F 'include "/etc/lighttpd/conf.d/pihole-admin.conf"' "${lighttpdConfig}"; then
+           sed -i '/include "\/etc\/lighttpd\/conf.d\/pihole-admin.conf"/d' "${lighttpdConfig}"
+        fi
```

Comment:
Do we need the `if` here?

```suggestion
        sed -i '/include "\/etc\/lighttpd\/conf.d\/pihole-admin.conf"/d' "${lighttpdConfig}"
```

Reply:
Possibly not, I was going for the reverse of what we currently to, so maybe too verbose.